pkg/cmd/repo/edit/edit.go

```diff
@@ -8,44 +8,71 @@ import (
 	"io"
 	"io/ioutil"
 	"net/http"
+	"strings"
 
+	"github.com/AlecAivazis/survey/v2"
 	"github.com/MakeNowJust/heredoc"
 	"github.com/cli/cli/v2/api"
 	"github.com/cli/cli/v2/internal/ghinstance"
 	"github.com/cli/cli/v2/internal/ghrepo"
 	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/cli/cli/v2/pkg/prompt"
 	"github.com/cli/cli/v2/pkg/set"
 	"github.com/spf13/cobra"
 	"golang.org/x/sync/errgroup"
 )
 
+const (
+	allowMergeCommits = "Allow Merge Commits"
+	allowSquashMerge  = "Allow Squash Merging"
+	allowRebaseMerge  = "Allow Rebase Merging"
+
+	optionAllowForking      = "Allow Forking"
+	optionDefaultBranchName = "Default Branch Name"
+	optionDescription       = "Description"
+	optionHomePageURL       = "Home Page URL"
+	optionIssues            = "Issues"
+	optionMergeOptions      = "Merge Options"
+	optionProjects          = "Projects"
+	optionTemplateRepo      = "Template Repository"
+	optionTopics            = "Topics"
+	optionVisibility        = "Visibility"
+	optionWikis             = "Wikis"
+)
+
 type EditOptions struct {
-	HTTPClient   *http.Client
-	Repository   ghrepo.Interface
-	Edits        EditRepositoryInput
-	AddTopics    []string
-	RemoveTopics []string
+	HTTPClient      *http.Client
+	Repository      ghrepo.Interface
+	IO              *iostreams.IOStreams
+	Edits           EditRepositoryInput
+	Topics          []string
```

Comment:
What is the purpose of the "Topics" field? "AddTopics" and "RemoveTopics" are self-explanatory, but adding "Topics" to the same level is vague and ambiguous. Furthermore, it turns out that this field isn't even populated by user input. Can we remove it?

Reply:
@mislav The `Topics` field definitely could be better named. It is supposed to represent the current topics of the repository. It is not user input, and is populated by the API. The reason I added it to the options struct was for ease of passing it around since it is necessary for the interactive flow but also again when  we actually set the repo topics. This just seemed like the cleanest way and fit the style of editing the opts for the interactive function.